tools/halide_image.h

```diff
@@ -184,6 +184,10 @@ class Image {
         return &(contents->buf);
     }
 
+    uint8_t *alloc() const {
+        return contents->alloc;
```

Comment:
I'm not sure this is something we should expose or report on... is it really that interesting? I think all it is is the allocation *if* the image allocated it. It seems rarely useful. data() is the thing that I'd expect to be more interesting.